app/src/main/java/org/simple/clinic/summary/PatientSummaryModel.kt

```diff
@@ -46,6 +46,9 @@ data class PatientSummaryModel(
   val isTeleconsultationEnabled: Boolean
     get() = currentFacility!!.config.teleconsultationEnabled == true
 
+  val isUserLoggedIn: Boolean
+    get() = userLoggedInStatus == User.LoggedInStatus.LOGGED_IN
```

Comment:
Does this fail if it gets called before the  user logged in status has been loaded?